### PR TITLE
Update all documentation and fix the CloudFront role.

### PR DESCRIPTION
**Pull request by EmlynK**

### CODE REVIEW

File: docs/roles/aws/aws_iam_saml.md [hunk 1]
```diff
@@ -69,7 +69,9 @@ This requires boto and AWS-CLI on the provisioning server.
 ```yaml
 aws_iam_saml:
   aws_profile: "{{ _aws_profile }}" # Boto profile to use for AWS connections
-  tags: {} # Dict of AWS tags to apply
+  region: "eu-west-2" # AWS region to use
+  tags: {} # List of AWS tags to apply
```
Comment (gregharvey) [inline docs/roles/aws/aws_iam_saml.md]: This is wrong, probably accepted the wrong bit in the PR, `tags: {}` is a dict but the comment says `List of AWS tags`.

File: roles/aws/aws_iam_saml/README.md [hunk 2]
```diff
@@ -70,7 +70,7 @@ This requires boto and AWS-CLI on the provisioning server.
 aws_iam_saml:
   aws_profile: "{{ _aws_profile }}" # Boto profile to use for AWS connections
   region: "eu-west-2" # AWS region to use
-  tags: {} # Dict of AWS tags to apply
+  tags: {} # List of AWS tags to apply
```
Comment (gregharvey) [inline roles/aws/aws_iam_saml/README.md]: Same as above, is a dict, not a list.

File: docs/roles/aws/aws_iam_saml.md (hunk 1 above)
Comment (EmlynK) [reply docs/roles/aws/aws_iam_saml.md]: Sorted!

File: roles/aws/aws_iam_saml/README.md (hunk 2 above)
Comment (EmlynK) [reply roles/aws/aws_iam_saml/README.md]: Also sorted!

**Review by gregharvey — APPROVED** [review]

**Pull request merged**